Dockerfile.build

```diff
@@ -0,0 +1,30 @@
+FROM node:11.15.0 AS run
+RUN apt-get update && apt-get install -y python3-pip
+
+
+FROM run AS build
+COPY . /app/
+WORKDIR /app/
+
+RUN npm install
+RUN npm run-script lint
+RUN npm run-script compile
+
+WORKDIR /app/web
+RUN npm ci
+RUN npm run build
+WORKDIR /app/
+RUN npm run-script dist
+
+FROM run
```

Comment:
For the last stage we should inherit from alpine like it was done before

Reply:
I don't know why we should. If it is tested in debian, we should run it in debian (actually in criteo we will run it in a centos image anyway eventually)

Reply:
Well ideally we should test the final image we run in prod, not a intermediary one. But it might be a bit complicated.

Reply:
I think it will be a next step since the image we should run in prod needs to be built internally. So we can't test with it.